src/lib/y2network/interface.rb

```diff
@@ -28,14 +28,46 @@ class Interface
     attr_accessor :description
     # @return [Symbol] Interface type
     attr_accessor :type
+    attr_reader :configured
+    attr_reader :hardware
+
+    # Shortcuts for accessing interfaces' ifcfg options
+    ["STARTMODE", "BOOTPROTO"].each do |ifcfg_option|
```

Comment:
The idea is that `Y2Network::Interface` remains backend-agnostic, and these lines are defeating that purpose. Moreover, this information should be part of `Y2Network::ConnectionConfig` classes. However, I guess that you need those classes to be finished before using them :) so, for the time being, please add a note here that this logic should be moved in the near future.

Reply:
I don't have functionality from underlying layers ... we currently use NetworkInterfaces as access to backend everywhere. This will have to be cleaned once we have generic layer for accessing backend.